packages/dataviews/src/components/dataviews-pagination/index.tsx

```diff
@@ -37,41 +58,32 @@ function DataViewsPagination() {
 				<HStack
 					justify="flex-start"
 					expanded={ false }
-					spacing={ 2 }
-					className="dataviews-pagination__page-selection"
+					spacing={ 1 }
+					className="dataviews-pagination__page-select"
 				>
-					{ createInterpolateElement(
-						sprintf(
-							// translators: %s: Total number of pages.
-							_x( 'Page <CurrentPageControl /> of %s', 'paging' ),
+					<div aria-hidden>{ __( 'Page' ) }</div>
```

Comment:
Why we removed `createInterpolateElement` here? It feels like the current approach for translation is not ideal.. I'm not 100% sure if the current approach is right, but I was wondering why it was needed? I'll cc @swissspidy if he has any suggestion.

Reply:
Yeah I wouldn't remove it. Doing something like `__( 'of %s' )` makes this worse. `createInterpolateElement` is important here to avoid such string concatenation. 

I would even improve the translatable string a little bit like this:

```
sprintf(
	/* translators: 1: Current page, 2: Total pages. */
	_x( '%1$s of %2$s', 'paging' ),
	'<CurrentPageControl />',
	totalPages
)
```

Reply:
My bad, fixed in [bf1d768](https://github.com/WordPress/gutenberg/pull/63815/commits/bf1d7684eff41d0151ef640a0c14ee9db9db6fb1).